pkg/upstream/transformer_test.go

```diff
@@ -13,6 +18,80 @@ func ns(path string) Namespace {
 	return strings.Split(path, "/")
 }
 
+func Test_NewRedirectTransform200(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if data, err := ioutil.ReadFile("testdata/fuse.html"); err == nil {
```

Comment:
by convention we orient this so that we check `err != nil`, flipping around the if/else blocks in this case. (the `data` variable is still in scope in the else block)